ignite/contrib/handlers/throughput_benchmark_wrapper.py

```diff
@@ -0,0 +1,244 @@
+import torch
+import contextlib
+from typing import Callable
+from ignite.engine import Events, Engine
+
+
+class ExecutionStats(object):
```

Comment:
Why can't we just import this class ?

Reply:
As far as I understood @justusschock previous comment about me importing `torch.utils.ThroughputBenchmark` was that this wouldn't work for older pytorch versions? And by loading the `ThroughputBenchmark` lazily I can't access it for typing globally.

I could've misunderstood what was meant by:
> Can you wrap this in a way, that you first check the pytorch version if it is recent enough? This will probably result in a more understandable error message then a plain import error

Reply:
Ok, I see, sorry I also missed that exchange.

> I could've misunderstood what was meant by:
> > Can you wrap this in a way, that you first check the pytorch version if it is recent enough? This will probably result in a more understandable error message then a plain import error

I think the idea is to do it like here : https://github.com/pytorch/ignite/blob/master/ignite/contrib/handlers/mlflow_logger.py#L233

Maybe we can also skip the problem of typing.

Reply:
And just saw that you already did like that.